iocore/net/quic/QUICStreamManager.h

```diff
@@ -40,6 +40,12 @@ class QUICStreamManager : public QUICFrameHandler, public QUICFrameGenerator
 {
 public:
   QUICStreamManager(QUICContext *context, QUICApplicationMap *app_map);
+  ~QUICStreamManager()
+  {
+    for (auto stream = stream_list.pop(); stream != nullptr; stream = stream_list.pop()) {
+      _stream_factory.delete_stream(stream);
+    }
+  }
```

Comment:
Probably best to declare `~QUICStreamManager()` here and define in the `.cc` file. That will be consistent with the rest of the functions here.